api/clients/Admin/src/main.js

```diff
@@ -230,7 +230,7 @@ function refreshView(){
 
                 if (sensor.installed_at) {
                     measurementsPs.push(new Promise(function (resolve) {
-                        api.getPlaceMeasurements({id: sensor.installed_at, types: ['wifi']})
+                        api.measurementsPlaces({ids: [sensor.installed_at], types: ['wifi']})
```

Comment:
humm the implicit convention here is that the one starting with get are for the classic queries but for complex queries there is no get (see prepareApi.js)